### DISCUSSION TITLE
CKAD Questions, Practice Exam, Exercises

### DISCUSSION THREAD

**Discussion by StevenJ09** [dgkanatsios/CKAD-exercises#221] [General]
I’d like to share some resources, exercises, practice questions, the practice exam for CKAD exam preparation that I personally found very helpful, especially the Study4Exam's CKAD practice exam.
[https://github.com/bmuschko/ckad-crash-course#additional-resources](https://github.com/bmuschko/ckad-crash-course#additional-resources)

**Comment by dgkanatsios** [comment]
thanks for sharing!!!

**Comment by saranyanvl** [comment]
This practice exam was highly useful in completing CKAD. Other than that it will be good to take a course targeting CKAD. 
Note that from September 28 2021..there are slight changes in CKAD curriculum. Compiled my experience and these details in https://youtu.be/h9-eBWeHaAA.

**Comment by jacklinesmiths377-cloud** [comment]
I really appreciate how **Dumpsforsure** made my **CKAD** preparation stress-free. The material was easy to follow, and the practice sessions helped me get exam-ready. My hard work paid off, and I’m proud to say I cleared it by passing from Dumpsforsure.”

**Reply by Greeley524** [reply]
Hey, I have also passed my CKAD exam from Dumpsforsure however i would also recommend for anyone check it out their authentic practice questions and achieved their goals. Thanks!

**Comment by mekilamac** [comment]
I recently prepared for the **CKAD** exam, and one of my friends passed it just last week. He recommended **Certs4Future** because the practice questions helped him understand the exam format and covered many of the key topics he encountered during his preparation.
